### PR TITLE
Integrity unit id3

### PR DESCRIPTION
**Pull request by OstapMelnychuk**

### CODE REVIEW

**Review by DmytroSavchuk — COMMENTED** [review]
1) (Lab1/src/com/company/Main.java) main() - it's will be good if you can create some method or class for testing your equationManager.areParallelLines() method.
2) (Lab1/src/com/company/ThreadCreator.java) - ThreadCreator, createAndRunThreads() - 25-28 rows, same like a first comment.
3) (Lab2/.../UserController.java and Others) - i think ID must be written with capital character.

**Review by DmytroSavchuk — COMMENTED** [review]
HardCode it's my Live(Bon Jovi - It's My Life )

**Review by 0lezhka — APPROVED** [review]

File: Labs/BestFriendsIntegrityUnitID 3/Skorohoda/Lab2/src/main/java/com/example/demo/controller/TeamController.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        this.teamMapper = teamMapper;
+    }
+
+    @ApiOperation(value = "Find all teams")
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = HttpStatuses.OK, response = TeamDTO.class)
+    })
+    @GetMapping
+    public ResponseEntity<List<TeamDTO>> findAll() {
+        return ResponseEntity.status(HttpStatus.OK).body(teamService.findAll().stream().map(teamMapper::toDto)
```
Comment (0lezhka) [inline Labs/BestFriendsIntegrityUnitID 3/Skorohoda/Lab2/src/main/java/com/example/demo/controller/TeamController.java:35]: Can you move logic of converting entity to dto in service ?

Comment (OstapMelnychuk) [reply Labs/BestFriendsIntegrityUnitID 3/Skorohoda/Lab2/src/main/java/com/example/demo/controller/TeamController.java:35]: You think it should be there? Our tech-expert didn`t told us anything when he had reviewed our code.

**Pull request merged**